src/win/util.c

```diff
@@ -1380,6 +1380,12 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = uv_malloc(1);
+  if (pwd->gecos == NULL)
+    return UV_ENOMEM;
+
+  *(pwd->gecos) = '\0';
```

Comment:
I'm not sure I agree.  For one, we could substitute `gecos` with the full username in due time.  (I forgot what Windows calls it but I know it exists.)